src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -538,23 +544,22 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
      * Calculates the response frequency for each choice
      */
     private int[][] calculateResponseFrequency(
-            List<FeedbackResponseAttributes> responses,
-            FeedbackRubricQuestionDetails fqd) {
+             List<FeedbackResponseAttributes> responses, FeedbackRubricQuestionDetails fqd) {
```

Comment:
This might be better this way:
```java
private int[][] calculateResponseFrequency(List<FeedbackResponseAttributes> responses,
                                           FeedbackRubricQuestionDetails fqd) {
```